.projenrc.ts

```diff
@@ -115,13 +115,16 @@ const pipeline = new yarn.TypeScriptWorkspace({
 
   devDeps: [
     'eslint@^8',
+    `@aws-cdk/integ-runner@${cdkVersion}-alpha.0`,
+    `@aws-cdk/integ-tests-alpha@${cdkVersion}-alpha.0`,
     '@typescript-eslint/eslint-plugin@^7',
     '@typescript-eslint/parser@^7',
     '@typescript-eslint/typescript-estree@^7',
   ],
 
   peerDeps: [`cdk-nag@^${cdkNagVersion}`, `aws-cdk-lib@^${cdkVersion}`, `constructs@^${constructsVersion}`],
   bundledDeps: ['@cloudcomponents/cdk-pull-request-approval-rule', '@cloudcomponents/cdk-pull-request-check', 'yaml'],
+  deps: [`cdk-pipelines-github`],
```

Comment:
I recommend to also put the version there so you can change it from outside in case you need to pin it in the future (e.g: they stop supporting some feature)